scripts/upsert/courses.ts

```diff
@@ -0,0 +1,36 @@
+import { Term } from '../../lib/term';
+import { differenceInDays, differenceInMinutes } from 'date-fns';
+import { upsertCourses } from '../../lib/courses';
+import { findLatestTask, createTask } from '../../lib/task';
+
+export const upsertCoursesScript = async (term: string, registrationDay: Date, dropDate: Date) => {
+  const today = new Date();
+
+  const daysUntilRegistration = differenceInDays(registrationDay, today);
+
+  const lastUpdated = await findLatestTask(`upsertCourses-${term}`);
```

Comment:
I almost feel making a `term` column here might be good for the tasks table so? What do you think?

Reply:
Hmm that might be useful... if tasks are mostly related to term it'd be helpful to specify that. But is there a downside for using the string params instead?